### PR TITLE
Sass integration

### PR DESCRIPTION
**Pull request by wafaworks**
With @christianJCK we created sass files and some variables for colors and typography to use in the project. 
Tell us if everything's ok or if something's missing.

### CODE REVIEW

File: src/BlogBundle/Resources/views/Default/index.html.twig [hunk 1]
```diff
@@ -1 +1,8 @@
-Hello World!
```
Comment (ngdo-pro) [inline src/BlogBundle/Resources/views/Default/index.html.twig:1]: Can you remove this file please ?

**Review by ngdo-pro — CHANGES_REQUESTED** [review]

**Review by ngdo-pro — APPROVED** [review]

**Review by shog189 — APPROVED** [review]
Ok for me after changes :+1:

**Review by dbeauseigneur — APPROVED** [review]
Ok for me

**Review by Kaji-U — APPROVED** [review]
Ok :)

**Pull request merged**